ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/events/Raw.scala

```diff
@@ -42,13 +42,14 @@ private[events] sealed trait Raw[+E] {
 
 }
 
-private[events] object Raw {
+// TODO append-only: FIXME move
+object Raw {
 
   /** Since created events can be both a flat event or a tree event
     * we share common code between the two variants here. What's left
     * out is wrapping the result in the proper envelope.
     */
-  private[events] sealed abstract class Created[E](
+  sealed abstract class Created[E](
```

Comment:
Is opening up visibility for most of the classes/traits/objects a concious decision in this PR?

Reply:
yepp, this guy needs to be move to another place, probably storage...we need to figure out how. hence the todo at line 45.